doc/releases/changelog-0.25.0.md

```diff
@@ -700,6 +700,9 @@ of operators.
 
 <h3>Bug fixes 🐞</h3>
 
+* Fixed a bug where the parameter-shift rule wasn't defined for `qml.kUpCCGSD`.
```

Comment:
Thanks @isaacdevlugt for fixing this. Do you think we need a test or tests to check for such behaviour?

Reply:
Good call @soranjh, it would be good to add a test to check that the `qml.kUpCCGSD` template gives the correct derivative with both backprop and parameter-shift

Reply:
Okay! I can work on this